smriprep/workflows/outputs.py

```diff
@@ -64,7 +64,7 @@ def init_anat_reports_wf(reportlets_dir, freesurfer,
         name='ds_std_t1w_report', run_without_submitting=True)
 
     workflow.connect([
-        (inputnode, tf_select, [('template', 'template'),
+        (inputnode, tf_select, [(('template', _drop_cohort), 'template'),
```

Comment:
why not just connect from `split_desc.name` since that is dropping the cohort?

Reply:
name is not exposed outside the workflow - can we live with this inline?

Reply:
this connection is within workflow, no?
i'm suggesting changing it to 
```suggestion
        (split_desc, tf_select, [('name', 'template'),
```
to avoid an excess function call

Reply:
I can't find `split_desc` in this module... :D